### DISCUSSION TITLE
notes_graph.json

### DISCUSSION THREAD

**Discussion by andyw-0612** [maximevaillancourt/digital-garden-jekyll-template#142] [Q&A] [answered]
I removed the lines in `_layouts/default.html` to prevent the graphs of notes from generating. However, every time I run the site, a file called `notes_graph.json` is still generating in the `_includes` directory, which I assume to be related to the graphs. Is it possible to prevent the file from generating? Thanks!

**Comment by maximevaillancourt** [comment, marked as answer]
To actually stop generating the file, comment out these lines: https://github.com/maximevaillancourt/digital-garden-jekyll-template/blob/76ac51bf7696c6b2a02852a2578666a51f393cb6/_plugins/bidirectional_links_generator.rb#L102-L105

To prevent Jekyll running into an infinite loop locally (if that's happening to you), add this line to your project: https://github.com/maximevaillancourt/digital-garden-jekyll-template/blob/76ac51bf7696c6b2a02852a2578666a51f393cb6/_config.yml#L3
